service/node.go

```diff
@@ -286,20 +286,22 @@ func (s *service) NodeGetCapabilities(
 	*csi.NodeGetCapabilitiesResponse, error,
 ) {
 	capabilities := []*csi.NodeServiceCapability{
-		{
+		// TODO - Need to return this capability when Stage/Unstage volume logic is implemented
+		/*{
```

Comment:
I think we can remove the unsupported capabilities instead of keeping them commented.

Reply:
Removed. Thanks!